pkg/binlog/position_test.go

```diff
@@ -373,22 +374,9 @@ func (t *testPositionSuite) TestCompareCompareLocation(c *C) {
 			},
 			"",
 			0,
-		}, {
-			// pos1 = pos2
-			gmysql.MariaDBFlavor,
-			gmysql.Position{
-				Name: "binlog.00001",
-				Pos:  123,
-			},
-			"",
-			gmysql.Position{
-				Name: "binlog.00001",
-				Pos:  123,
-			},
-			"",
```

Comment:
Why do we delete this?

Reply:
I think it is duplicate with the test case above, but in fact they have different flavor. I add it back in 45c9f69